sdcm/logcollector.py

```diff
@@ -814,6 +814,27 @@ def collect_logs(self, local_search_path=None) -> list[str]:
         return super().collect_logs(local_search_path)
 
 
+def save_kallsyms_map(node):
+
```

Comment:
Why not create this function as method of Base node?

Reply:
this is something at this point very specific to DB nodes